src/Process/3dTilesProcessing.js

```diff
@@ -138,15 +138,17 @@ export function init3dTilesLayer(context, layer) {
     });
 }
 
+function setDisplayed(node, display) {
+    if (node.content) {
```

Comment:
You should copy-paste a comment here explaining why your not simply doing `node.material.visible = false`